app/views/patients/show.html.erb

```diff
@@ -71,8 +74,34 @@
         <% end %>
       </div>
     </div>
-
+    <div class="tile is-ancestor">
```

Comment:
Can you add all of this to `app/views/health_cards/show.pdf.erb` so that lab results populate onto the PDF too? (let me know if there are any issues with this)

Reply:
resolved